app/styles/modules/_settings.scss

```diff
@@ -70,7 +93,10 @@ body.settings #main-content.card {
 
     @include respond-to('small') {
       margin-top: 9px;
-      width: 29%;
+      html[dir=ltr] & {
+    	 width: 29%;
+     	}
```

Comment:
This change to avoid text overflow the sing out button, just like this:

![2016-05-25 13-40-45](https://cloud.githubusercontent.com/assets/9789109/15541543/a2455794-2295-11e6-90ab-4f31640e9ee7.png)
![2016-05-25 13-52-53](https://cloud.githubusercontent.com/assets/9789109/15541560/b88878b0-2295-11e6-8255-99bcd6c28085.png)